backend/models/projects/projects-model.test.js

```diff
@@ -11,7 +27,133 @@ afterAll(async () => await db.closeDatabase())
+            expect(expectedEndDate).toEqual(retrievedProject.end_date)
+            expect(expectedDiscordUrl).toEqual(retrievedProject.discord_url)
+            expect(expectedGithubUrl).toEqual(retrievedProject.github_url)
+            expect(expectedDemoUrl).toEqual(retrievedProject.demo_url)
+            expect(expectedCreatedAt).toEqual(retrievedProject.created_at)
+            expect(expectedUpdatedAt).toEqual(retrievedProject.updated_at)
+        })
+
+        test("Add a Project to Database", async () => {
+            const project = new ProjectsModel(testProject1)
+
+            const createdProject = await project.save()
+
+            // Expected value set
+            const expectedTitle = testProject1.title
+            const expectedStatus = testProject1.status
+            const expectedDescription = testProject1.description
+            const expectedStartDate = testProject1.start_date
+            const expectedEndDate = testProject1.end_date
+            const expectedDiscordUrl = testProject1.discord_url
+            const expectedGithubUrl = testProject1.github_url
+            const expectedDemoUrl = testProject1.demo_url
+            const expectedCreatedAt = testProject1.created_at
+            const expectedUpdatedAt = testProject1.updated_at
+
+            // Tests
+            expect(createdProject.title).toEqual(expectedTitle)
+            expect(createdProject.status).toEqual(expectedStatus)
+            expect(createdProject.description).toEqual(expectedDescription)
+            expect(createdProject.start_date).toEqual(expectedStartDate)
+            expect(createdProject.end_date).toEqual(expectedEndDate)
+            expect(createdProject.discord_url).toEqual(expectedDiscordUrl)
+            expect(createdProject.github_url).toEqual(expectedGithubUrl)
+            expect(createdProject.demo_url).toEqual(expectedDemoUrl)
+            expect(createdProject.created_at).toEqual(expectedCreatedAt)
+            expect(createdProject.updated_at).toEqual(expectedUpdatedAt)
+        })
+
+        test("Update a Project in Database", async () => {
+            const project = new ProjectsModel(testProject1)
+            await project.save()
+            const retrievedProject = await ProjectsModel.findOne({
+                title: "Project 1",
+            })
+
+            // Update values
+            retrievedProject.title = "Project 1 Updated"
+            retrievedProject.status = "Completed"
+            retrievedProject.description = "Updated Description"
+            retrievedProject.start_date = new Date("12-24-2022")
+            retrievedProject.end_date = new Date("12-24-2023")
+            retrievedProject.discord_url = "https://discord.gg/12345"
+            retrievedProject.github_url = "github.com/12345"
+            retrievedProject.demo_url = "https://demo.com/12345"
+            retrievedProject.created_at = new Date("09-27-2022")
+            retrievedProject.updated_at = new Date("09-30-2022")
```

Comment:
It is not necessary to re-assign the same data as the `testProject1` because we check if some of value in the fields were changed, and some others none. Recommend to use different values or remove all these assignments.